modules/dev.js

```diff
@@ -0,0 +1,18 @@
+import { Employee } from "./employee.js";
+
+export class Dev extends Employee {
+  #jobtype = "Dev";
+
+  constructor(id, salary, jobtype) {
+    super(id, salary);
+    this.#jobtype = jobtype;
+  }
+
+  getJobType() {
+    return "I am a Dev";
```

Comment:
```suggestion
    return `I am a ${this.#jobtype}`;
```
В тебе є приватний метод, який зберігає jobtype, треба просто повернути його